client/src/components/MainPage/QuestionsPage/QuestionCard/style.css

```diff
@@ -28,8 +36,12 @@ a {
   color: rgba(0, 0, 0, 0.38);
 }
 
-.box__card--content--box {
-  display: grid;
-  grid-template-columns: 1fr 1fr;
-  grid-gap: 10px;
+.box__card--content > :nth-child(2) {
```

Comment:
@HemaSAli I'd suggest adding a className to that second `div` and using that className here, instead of the nth-child selector.  That way, in the future if you rearrange the order of  `.box__card--content--box`, your styles here will still be applied to the correct element.

Reply:
thanks Tom , I edited it